apps/wallet/src/ui/app/staking/stake/StakeForm.tsx

```diff
@@ -21,30 +22,45 @@ export type StakeFromProps = {
     submitError: string | null;
     coinBalance: bigint;
     coinType: string;
+    coinDecimals: number;
     epoch: string;
     onClearSubmitError: () => void;
 };
 
+function AvailableBalance({ amount }: { amount: bigint }) {
+    const [formatted, symbol] = useFormatCoin(amount, SUI_TYPE_ARG);
```

Comment:
How do you know that amount is in SUI? Given that `StakeForm` accepts `coinType` to know the type of the coin either we need to use that here as well or remove that requirement from everywhere if we know that we only are going to support staking for SUI.